code/modules/vending/_vending.dm

```diff
@@ -636,10 +641,8 @@
 					freebie(user, 1)
 				if(26 to 75)
 					return
-				if(76 to 90)
+				if(76 to 100)
 					tilt(user)
-				if(91 to 100)
-					tilt(user, crit=TRUE)
```

Comment:
If i'm not mistaken the prior "crit chance" was 9% and the new "crit chance" is 15%, is that intended
I assume it is because there's less debilitating crits now but just double checking

Reply:
I tried to preserve the original crit chance value - it wasn't actually 9% per tilt, it was 9% per each roll

Whether or not I got the actual math right is beyond me, but I'm pretty sure it wasn't 9% in the grand scheme of thing

If someone knows math better than me please correct me because I am a goddamn idiot